### ISSUE TITLE
Formatting issues

### ISSUE THREAD

**Issue by maoo** [finos/calendar#73]
Below is a list of screenshots that show formatting issues on some calendar events; for each of those, we'd need to understand why they're rendering out differently than others.

![image (2)](https://github.com/finos/calendar/assets/327285/533f7b68-9971-488c-8267-70fc8cd34148)
![image (3)](https://github.com/finos/calendar/assets/327285/d512bc2b-3042-4154-a3dd-fe1f032eba53)
![image](https://github.com/finos/calendar/assets/327285/52ebefbc-7431-46a5-9eb7-b54bd9bf5430)

**Comment by gautamjuyal** [comment]
I figured what the problem was. The links appearing as plain text were rendered that way, since we are using html-react-parser, and the parser will render even links as plain text until we explicitly enclose the links in <a> tag. Other links which were working, used html syntax for formatting that's why the the links were rendered as hyperlinks. In case of some files where formatting issue occured, simply had links only and no html syntax. 
I simply tried it on some events, and it worked. I am trying to find a workaround, and thought I should let you know what exactly the problem was.
